88920ff33b860feebd2f7f97fd6200fa.svg

```diff
@@ -0,0 +1 @@
73933-1.50491-7.7962-.82548-11.60323.03181-3.5883.808-7.718,2.006-11.29266.49665-2.64516-1.1169-4.74985-4.635-2.84718-7.23211,1.1422-1.5591-1.45984-3.05738-2.59041-1.51415Z" transform="translate(-246.40761 -180.31644)" fill="#2f2e41"/><path d="M566.94844,719.6636h-216a1,1,0,0,1,0-2h216a1,1,0,0,1,0,2Z" transform="translate(-246.40761 -180.31644)" fill="#3f3d56"/><path d="M951.75743,412.75882H714.42736a1.835,1.835,0,1,1,0-3.66992H951.75743a1.835,1.835,0,1,1,0,3.66992Z" transform="translate(-246.40761 -180.31644)" fill="#e6e6e6"/><path d="M850.8673,311.50638H758.5775a19.47209,19.47209,0,0,1-19.45019-19.4502v-92.29a19.472,19.472,0,0,1,19.45019-19.4497h92.2898a19.472,19.472,0,0,1,19.45019,19.4497v92.29A19.47209,19.47209,0,0,1,850.8673,311.50638Z" transform="translate(-246.40761 -180.31644)" fill="#6c63ff"/><path d="M907.60729,354.30618h-92.2898a19.472,19.472,0,0,1-19.45019-19.4497v-92.29a19.47209,19.47209,0,0,1,19.45019-19.4502h92.2898a19.47209,19.47209,0,0,1,19.45019,19.4502v92.29A19.472,19.472,0,0,1,907.60729,354.30618Z" transform="translate(-246.40761 -180.31644)" opacity="0.1" style="isolation:isolate"/><path d="M748.46813,465.6636a1,1,0,0,1-1-1v-18a1,1,0,0,1,2,0v18A1.00005,1.00005,0,0,1,748.46813,465.6636Z" transform="translate(-246.40761 -180.31644)" fill="#ccc"/><path d="M829.96813,465.6636a1,1,0,0,1-1-1v-18a1,1,0,0,1,2,0v18A1.00005,1.00005,0,0,1,829.96813,465.6636Z" transform="translate(-246.40761 -180.31644)" fill="#3f3d56"/><path d="M836.96813,465.6636a1,1,0,0,1-1-1v-18a1,1,0,0,1,2,0v18A1.00005,1.00005,0,0,1,836.96813,465.6636Z" transform="translate(-246.40761 -180.31644)" fill="#3f3d56"/><path d="M763.1483,465.98929a2.31355,2.31355,0,0,1-1.1604-.31494l-11.8938-8.02149c-.02246-.01367-.04467-.02734-.06665-.0415a2.26673,2.26673,0,0,1-1.05932-1.94776,2.29617,2.29617,0,0,1,1.16015-2.00927l11.80054-7.96485a2.31881,2.31881,0,0,1,3.53931,1.97217v16.00391a2.32917,2.32917,0,0,1-2.31983,2.32373Zm-.10083-2.00977a.3061.3061,0,0,0,.2605-.03662.31055.31055,0,0,0,.16016-.27734V447.66165a.32.32,0,0,0-.47974-.27734l-11.801,7.96533c-.19848.11719-.21923.24512-.21923.314a.30948.30948,0,0,0,.15966.27686l.05957.03711Z" transform="translate(-246.40761 -180.31644)" fill="#ccc"/><path d="M916.46813,465.6636a1,1,0,0,1-1-1v-18a1,1,0,0,1,2,0v18A1.00005,1.00005,0,0,1,916.46813,465.6636Z" transform="translate(-246.40761 -180.31644)" fill="#ccc"/><path d="M901.76988,465.98489a2.27232,2.27232,0,0,1-1.14135-.31,2.2964,2.2964,0,0,1-1.1604-2.00928V447.66165a2.31989,2.31989,0,0,1,3.48022-2.00879l11.91919,8.03858a2.26215,2.26215,0,0,1,1.10059,1.97216,2.2955,2.2955,0,0,1-1.126,1.98926l-11.83472,7.98487A2.4586,2.4586,0,0,1,901.76988,465.98489Zm-.30175-2.31933a.31994.31994,0,0,0,.47973.27734l11.846-7.99414a.33732.33732,0,0,0,.17432-.28418v-.001a.30949.30949,0,0,0-.15967-.27685l-11.91968-8.03906a.29746.29746,0,0,0-.2605.03662.31055.31055,0,0,0-.16015.27734Z" transform="translate(-246.40761 -180.31644)" fill="#ccc"/><circle cx="534.06052" cy="230.60742" r="12" fill="#3f3d56"/></svg>
```

Comment:
Not sure what this is? Want to double check it was supposed to be added and not in error

Reply:
Hey Kathy, thanks for checking in - & yes that's right, that's the home page image! SVG's are so cool because they're images made from code!

Reply:
& you can potentially manipulate the image and make it dynamic since it's code ^_^

Reply:
ooo cool!